packages/dbml-parse/src/lib/analyzer/validator/elementValidators/note.ts

```diff
@@ -50,7 +76,7 @@ export default class NoteValidator implements ElementValidator {
   }
 
   private validateSettingList(settingList?: ListExpressionNode): CompileError[] {
-    if (settingList) {
+    if (settingList && !(this.declarationNode.parent instanceof ProgramNode)) {
```

Comment:
Since we haven't support note color yet, we should prevent user from note's settings this since it currently have no effect. Suggest to remove this change.

Reply:
Removed